ODLA/platforms/tensorrt/odla_tensorrt.cc

```diff
@@ -300,6 +303,16 @@ static int64_t GetTotalElements(const odla_value_shape& dims) {
 }
 
 const int nvinfer1::Dims::MAX_DIMS;
+static std::string gen_str(const nvinfer1::Dims& d) {
```

Comment:
guard with some macro like 
```#ifndef NDEBUG```
?

Reply:
ok

Reply:
btw, could you squash into single commit ?

Reply:
When merging the PR into the master, squash will be the automatic choice. Any benefit to squash commits manually?

Reply:
I can enable "auto-merge" if it's already squashed

no significant benefits.